src/configs/all.ts

```diff
@@ -88,6 +88,7 @@ export const rules = {
     // Functionality
     "await-promise": true,
     // "ban": no sensible default
+    "ban-comma-operator": true,
```

Comment:
IMO this is also a good candidate for `latest.ts`

Reply:
Done.